spec/models/condition_spec.rb

```diff
@@ -1,5 +1,26 @@
 describe Condition do
   describe ".subst" do
+    context "evaluation of virtual custom attributes from left and right side" do
+      let(:custom_attribute_1)         { FactoryGirl.create(:custom_attribute, :name => "attr_1", :value => 20) }
+      let(:custom_attribute_2)         { FactoryGirl.create(:custom_attribute, :name => "attr_2", :value => 30) }
+      let(:name_of_custom_attribute_1) { "VmOrTemplate-#{CustomAttributeMixin::CUSTOM_ATTRIBUTES_PREFIX}attr_1" }
+      let(:name_of_custom_attribute_2) { "VmOrTemplate-#{CustomAttributeMixin::CUSTOM_ATTRIBUTES_PREFIX}attr_2" }
+      let(:vm)                         { FactoryGirl.create(:vm, :memory_reserve => 10) }
```

Comment:
yes, it does not need to, but I moved adding custom attributes to let a remove .save.

also added some other cases 

thanks 👍